pango/plan.sh

```diff
@@ -10,16 +10,18 @@ pkg_description="Pango is a library for laying out and rendering of text, with a
 pkg_deps=(
   core/bzip2
   core/cairo
-  core/coreutils
   core/expat
   core/fontconfig
   core/freetype
+  core/gcc-libs
   core/glib
   core/glibc
   core/harfbuzz
   core/libffi
+  core/libice
```

Comment:
Do you remember why we need to add this and libsm?

Reply:
IIRC this is because there's a link that wasn't satisfied in one of the built binaries.